app/models/game.rb

```diff
@@ -1,36 +1,40 @@
 class Game < ApplicationRecord
+  # belongs_to :black_player, class_name: 'Player'
   has_many :pieces
   has_many :players
-  scope :available, -> {where("total_players = 1")}
+  scope :available, -> { where('total_players = 1') }
   belongs_to :user
   after_create :current_user_is_white_player
   after_create :populate
 
+  # to initialize each game with the white_player as the user who created the game
+  # white_player_id needs to exist in the database
 
-  #to initialize each game with the white_player as the user who created the game 
-  #white_player_id needs to exist in the database
-  
-  #we need this for everything else to work
+  # we need this for everything else to work
   def square_occupied?(x_current, y_current)
-    pieces.active.where({x: x_current, y: y_current}).any? ? true : false
+    pieces.active.where(x: x_current, y: y_current).any? ? true : false
```

Comment:
Does this do the exact same thing with or without the `{ }`? I know with `{ }` it was checking the case where `x: x_current` AND `y: y_current` are true. As long as it's still checking both, and not "either," I am fine with this change. I just want to be sure!

Reply:
changed back!